### DISCUSSION TITLE
How to increase expression pedal amplitude

### DISCUSSION THREAD

**Discussion by newVideophile** [GrandOrgue/grandorgue#958] [Help] [answered]
I am using GrandOrgue on Linux (on a Raspberry Pi 4) with the Friesach sampleset. I am very happy with it except for one thing: the expression pedal amplitude is too low, i.e. I find that the volume is too high when the expression pedal is at its minimum position. 

Is there a way to adjust that ?

**Comment by jason-barba** [comment, marked as answer]
I am not sure if this is the right answer, but check https://github.com/GrandOrgue/grandorgue/discussions/769.

**Comment by newVideophile** [comment]
Yes, thank you!

It almost worked: I edited my ODF, I changed the AmpMinimumLevel paramter to a lower value, and started GO.

Then I got an error message (I translate to English, so the actual English message may be a bit different) : "The ODF does not correspond to the combination file. Importing could cause problems. Do you really want to import it?". I answered yes and everything went fine. And, most importantly, the parameter change was taken into account. 

Now, what should I do to avoid this error message? And what is a "combination file"?

**Reply by jason-barba** [reply]
As I understand, the combination files stores your combinations and generals for the sample set named in the ODF. If you change/edit the ODF GO will notice that it is not the same ODF that was used creating the combinations file and warns you.

For smaller changes like the one you did with AmpMinimumLevel it is emperically found no problem to answer YES. I guess major problems can occur if you delete a stop or something like that.

Good practise is to export settings and combinations before you do any changes. Then you can import them back if disaster hits.

**Reply by newVideophile** [reply]
I did some research to find out what a combination file is, and your explanation is right. 

But I don't want to have to click a button each time I start GO. This is a no go for me, as I use GO on a headless Raspberry Pi. 

By chance, I found an easy workaround: I just save the edited ODF under a different name. No more error message, problem solved.

Many thanks for your help!

**Reply by jason-barba** [reply]
When I have made changes to the ODF I use, I only have to answer the problem message once.

**Reply by newVideophile** [reply]
Not me, I always get this annoying message :-(

**Reply by oleg68** [reply]
> Not me, I always get this annoying message :-(

Export the combination again. The next time you won't get the message.

**Reply by newVideophile** [reply]
It didn't work for me. However saving the parameters did!

**Comment by ahall41** [comment]
The alternative is to adjust the midi settings. My pedal ranges from 55 to 127 (lower to upper limit). Change the 55 to 45 or 50 will have the same effect. (No good, I appreciate if it shows 0 - 127, of course).

**Reply by jason-barba** [reply]
I most humbly allow my self to dispute. Will not the pedal still deliver MIDI messages from 0 -127? If so, one quarter, range 0-55, of the possible pedal movement does not affect the output volume.

By changing the AmpMinimumLevel the full pedal mechanical movement will be allocated to a smaller dynamic volume range.

Please correct me if I am in err.

**Reply by ahall41** [reply]
My swell pedal fully open transmits 127, fully closed 55. So I can (and have) reduced the swell effect by changing this. Its an alternative to fiddling with the ODF **if** your midi input allows it. 
I found this out, because I first used listen for event (defaulting to 0-127) and wondered why the swell had so little effect.

**Reply by jason-barba** [reply]
Aha. I kind of misunderstood.
